### PR TITLE
Replace static deployment files with template yamls

### PR DESCRIPTION
**Pull request by hochiw**
- Use yaml templates to handle app configs
- Added checksum annotation for handle pod restarts on config change

Tested by deploying to local k8s, didn't test backend integrity as I didn't host a db 

## Will need to update deployment values before merging

refs oursky/likedao#265

### CODE REVIEW

**Review by elise-ng — APPROVED** [review]

File: deploy/likedao/templates/bdjuno.config.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  config.yaml: |
+    chain:
+        bech32_prefix:
+            {{- range .Values.bdjuno.config.chain.bech32Prefix }}
+              - {{ . }}
+            {{- end}}
+        modules:
+            {{- range .Values.bdjuno.config.chain.modules }}
+              - {{ . }}
+            {{- end}}
```
Comment (rickmak) [inline deploy/likedao/templates/bdjuno.config.yaml:15]: I don't think we will config modules?

File: deploy/likedao/templates/bdjuno.config.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        port: {{ .Values.bdjuno.config.database.port }}
+        user: {{ .Values.bdjuno.config.database.user }}
+        password: {{ .Values.bdjuno.config.database.password }}
+        schema: {{ .Values.bdjuno.config.database.schema }}
+        max_open_connections: {{ .Values.bdjuno.config.database.maxConnections }}
+        max_idle_connections: {{ .Values.bdjuno.config.database.maxConnections }}
+        partition_size: {{ .Values.bdjuno.config.database.partitionSize }}
+        partition_batch: {{ .Values.bdjuno.config.database.partitionBatch }}
+    logging:
+        level: debug
```
Comment (rickmak) [inline deploy/likedao/templates/bdjuno.config.yaml:45]: logging level need to be config.

File: deploy/likedao/templates/react-app.config.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+              "link": "{{ .link }}"
+            }
+          {{- end}}
+        {{- end}}
+      ],
+      footerLinks: {
+        osmosis: "{{ .Values.reactApp.footerLinks.osmosis }}",
+        liquid: "{{ .Values.reactApp.footerLinks.liquid }}",
+        emeris: "{{ .Values.reactApp.footerLinks.emeris }}",
+        contactSupport: "{{ .Values.reactApp.footerLinks.contactSupport }}",
```
Comment (rickmak) [inline deploy/likedao/templates/react-app.config.yaml:46]: I think we should let the whole footerLinks configurable. Not making sense for oither chains just fill in osmosis or liquid link?

Most of the cosmos chain token are not listed on liquid.

**Review by rickmak — CHANGES_REQUESTED** [review]

File: deploy/likedao/templates/bdjuno.config.yaml (hunk 1 above)
Comment (hochiw) [reply deploy/likedao/templates/bdjuno.config.yaml:15]: it's possible for other chains to add in their own modules, we will need to add `iscn` to this field when it's ready as well

File: deploy/likedao/templates/react-app.config.yaml (hunk 3 above)
Comment (hochiw) [reply deploy/likedao/templates/react-app.config.yaml:46]: i think we can keep contactSupport as is but let's make the `getToken` section an array of object with name and link field

File: deploy/likedao/templates/bdjuno.config.yaml (hunk 1 above)
Comment (rickmak) [reply deploy/likedao/templates/bdjuno.config.yaml:15]: I think we require some moudles in order for dao to work?

Let's make it only for addiational modules, not listing all.

**Comment by hochiw** [comment]
@rickmak updated, tested frontend by deploying to local k8s

**Comment by rickmak** [comment]
Merged and pushed, need to update the likedao-deploy repos.

**Pull request merged**